R/tab_create_modify.R

```diff
@@ -351,9 +389,17 @@ tab_spanner_delim <- function(data,
 #' @import rlang
 #' @export
 tab_row_group <- function(data,
-                          group = NULL,
+                          label = NULL,
                           rows = NULL,
-                          others = NULL) {
+                          id = label,
+                          others_label = NULL) {
```

Comment:
Can we change the `other_label` default to `missing_arg`, and allow users to unset `others_label` by explicitly passing `NULL`?

Reply:
Or perhaps `others_label` goes into tab options as `default_row_group_label` or something

Reply:
The second option was implemented (i.e., using `tab_options()` to set the default row group label).